index.js

```diff
@@ -0,0 +1,7 @@
+const {server} = require('./server.js');
+
+const port = process.env.PORT || 3300;
+
+server.listen(port, () => {
+    console.log('\n Server listening on port ${port}');
```

Comment:
If You want to use ES6 template strings You need to wrap up that part in backticks, not single quotes.